### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by Tagadda — APPROVED** [review]
Thanks :D

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    [resources.install_dir]
+
+    [resources.data_dir]
+    subdirs =["Podcasts", "Playlists"]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "openjdk-11-jre ffmpeg"
```
Comment (Tagadda) [inline manifest.toml:63]:
```suggestion
    packages = "ffmpeg"
    packages_from_raw_bash = """
    debian=$(lsb_release --codename --short)
    if [[ $debian = "bullseye" ]]; then
        echo "openjdk-11-jre";
    elif [[ $debian = "bookworm" ]]; then
        echo "openjdk-17-jre";
    fi
    """
```

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**